common/app/renderers/DotcomRenderingService.scala

```diff
@@ -392,6 +393,16 @@ class DotcomRenderingService extends GuLogging with ResultWithPreconnectPreload
     val json = DotcomRenderingDataModel.toJson(dataModel)
     post(ws, json, Configuration.rendering.articleBaseURL + "/Article", CacheTime.Facia)
   }
+
+  def getCrossword(
+      ws: WSClient,
+      crosswordPage: CrosswordPageWithContent,
+      pageType: PageType,
+  )(implicit request: RequestHeader): Future[Result] = {
+    val dataModel = DotcomRenderingDataModel.forCrossword(crosswordPage, request, pageType)
+    val json = DotcomRenderingDataModel.toJson(dataModel)
+    post(ws, json, Configuration.rendering.articleBaseURL + "/Article", CacheTime.Facia)
```

Comment:
Non blocking comment for this PR: When you pick up the work to implement crosswords in DCR we will need to re-think this endpoint and the base url. 

The WebX team is currently under the process of [spinning up multiple AWS stacks](https://github.com/guardian/dotcom-rendering/issues/8351) to split the traffic. Atm we have one stack for articles and one for the rest of the content rendered by DCR (fronts & interactives):

```mermaid
graph LR
    Request-->Fastly
    Fastly-->Router
    Router-->FEArticleLB
    Router-->FEFaciaLB
    Router-->FEApplicationsLB
    subgraph Frontend
        FEArticleLB-->FEArticleApp
        FEFaciaLB-->FEFaciaApp
        FEApplicationsLB-->FEApplicationsApp
    end
    FEArticleApp--/Article-->DcrArticleLB-->DcrArticleApp
    FEFaciaApp--/Front-->DcrRenderingLB-->DcrRenderingApp
    FEApplicationsApp--/Interactives-->DcrRenderingLB-->DcrRenderingApp
```

Which stack handles the traffic is decided by the base url, e.g.
[`Configuration.rendering.articleBaseURL`](https://github.com/guardian/frontend/blob/1d827d0941bd0b55632aa05e23858fb28dc40a88/common/app/renderers/DotcomRenderingService.scala#L235C61-L235C65) --> Article app
[`Configuration.rendering.baseURL`](https://github.com/guardian/frontend/blob/main/common/app/renderers/DotcomRenderingService.scala#L310) --> Rendering app

Crosswords are in `applications` app in frontend, so it looks like traffic for those will be sent to the [same app as interactives](https://github.com/guardian/dotcom-rendering/issues/9325). 

So your endpoint will be sth like:

* base url --> The correct base url at the time
* endpoint --> You will need to add a [`/Crossword` endpoint in DCR](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/server/server.prod.ts#L64-L72)

That's all for the future, just wanted to give a heads up - this PR looks good to me!

Reply:
That's incredibly helpful info, thank you!